extensions/wikia/VideoHandlers/apiwrappers/ApiWrapper.class.php

```diff
@@ -900,8 +900,7 @@ protected function getVideoName() {
  * those are only used by video migration scripts and serve no other purpose
  */
 abstract class LegacyVideoApiWrapper extends PseudoApiWrapper {
-	//@todo change this url
-	static $THUMBNAIL_URL = 'http://community.wikia.com/extensions/wikia/VideoHandlers/images/NoThumbnailBg.png';
+	static $THUMBNAIL_PATH = '/extensions/wikia/VideoHandlers/images/NoThumbnailBg.png';
```

Comment:
I wonder if we should load this from the nocookie URL instead, what do you think? i.e. https://slot1-images.wikia.nocookie.net/__cb1510912377/common/extensions/wikia/VideoHandlers/images/NoThumbnailBg.png